### PR TITLE
Updated and added more tests to test_flow_handler.py

### PR DESCRIPTION
**Pull request by Sekhar-Kumar-Dash**
## Changes proposed

This pull request adds more tests for flow_handler.py

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: tests/test_flow_handler.py [hunk 1]
```diff
@@ -23,3 +27,294 @@ def test_is_supported_flow_without_ts(
...
+
+
+# testing handle_weird
+def test_handle_weird(flow, mock_db):
+    flow_handler = FlowHandler(mock_db, None, flow)
+    flow_handler.profileid = "profile_id"
+    flow_handler.twid = "timewindow_id"
+    flow_handler.handle_weird()
+
+    mock_db.publish.assert_called_with("new_weird", ANY)
```
Comment (AlyaGomaa) [inline tests/test_flow_handler.py:118]: Hello @Sekhar-Kumar-Dash 
why are you using ANY? insead of passing the flow as a dict? i think that would be better

**Review by AlyaGomaa — COMMENTED** [review]
Hello @Sekhar-Kumar-Dash 
I see your code still has the issue of long lines. max line chars should be 78 chars.
Can you please configure your IDE to user PEP8 style guide?
Thanks for your PR! i will re-review again when you fix all calls to cassert_called_with() with ANY parameter.

**Comment by Sekhar-Kumar-Dash** [comment]
> Hello @Sekhar-Kumar-Dash I see your code still has the issue of long lines. max line chars should be 78 chars. Can you please configure your IDE to user PEP8 style guide? Thanks for your PR! i will re-review again when you fix all calls to cassert_called_with() with ANY parameter.

hey @AlyaGomaa  PYcharm is showing just one PEP8 style error  
image
are getting more error in your  IDE

File: tests/test_flow_handler.py (hunk 1 above)
Comment (Sekhar-Kumar-Dash) [reply tests/test_flow_handler.py]: Hey @AlyaGomaa  just fixed it please take a look

**Comment by AlyaGomaa** [comment]
Hello @Sekhar-Kumar-Dash. checkingg

**Comment by AlyaGomaa** [comment]
Hey @Sekhar-Kumar-Dash thanks for the awesome PR! we appreciate it!

**Pull request merged**